Source/UnitTests/Core/DSP/DSPTestBinary.h

```diff
@@ -0,0 +1,56 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+
+#include <array>
+
+static const std::vector<u16> s_dsp_test_bin = {
```

Comment:
The definition for these should be placed in a cpp file, which is then `extern const` exposed in a header. Also the `<array>` include isn't being used here, if that was intended to be used instead of a vector.